### PR TITLE
Use old course 6 numbers for HKN

### LINKED ISSUE #10: change the hkn search to use old numbers
we should change the hkn search to use old numbers

### PR DESCRIPTION
**Pull request by SamZhang3**
Use old course 6 course numbers when available for links to HKN Underground Guide.

Fixes #10.

### CODE REVIEW

File: src/lib/class.ts [hunk 1]
```diff
@@ -418,9 +418,10 @@ export class Class {
       extraUrls.unshift({ label: "More Info", url: this.rawClass.u });
     }
     if (this.course === "6") {
+      const number = this.oldNumber ? this.oldNumber : this.number;
```
Comment (cjquines) [inline src/lib/class.ts:421]: is this equivalent to `this.oldNumber ?? this.number`?

**Review by cjquines — APPROVED** [review]
lgtm!

File: src/lib/class.ts [hunk 2]
```diff
@@ -418,7 +418,7 @@ export class Class {
       extraUrls.unshift({ label: "More Info", url: this.rawClass.u });
     }
     if (this.course === "6") {
-      const number = this.oldNumber ? this.oldNumber : this.number;
+      const number = this.oldNumber ?? this.number;
       extraUrls.push({
         label: "HKN Underground Guide",
         url: `https://underground-guide.mit.edu/search?q=${number}`,
```
Comment (gabrc52) [inline src/lib/class.ts:424]: could `this.oldNumber ?? this.number` go here directly?

Comment (cjquines) [reply src/lib/class.ts]: merge it in!!! i approved it!!! no one will stop you

**Pull request merged**